modules/web/src/assets/css/global/_theme.scss

```diff
@@ -216,6 +216,17 @@
     }
   }
 
+  .km-provider-edit-settings {
+    background-color: map.get($colors, option-background);
+    color: map.get($colors, text);
```

Comment:
i don't think `color` property is requried as color is applied based on ` .mat-menu-item` can you check whether it is required or not?